src/libcadet/model/binding/MobilePhaseModulatorLangmuirBinding.cpp

```diff
@@ -215,16 +236,31 @@ class MobilePhaseModulatorLangmuirBindingBase : public ParamHandlerBindingModelB
 			const double gamma = static_cast<double>(p->gamma[i]);
 			const double beta = static_cast<double>(p->beta[i]);
 			const double qMax = static_cast<double>(p->qMax[i]);
-			const double ka = static_cast<double>(p->kA[i]) * exp(gamma * yCp[0]);
+			const double ka = static_cast<double>(p->kA[i]);
+			double kaEGammaC0;
```

Comment:
I think best coding practice is to give an initial value here ``double kaEGammaC0`= 1.0;``, even though this shouldnt cause any issues.